src/test/java/org/sagebionetworks/bridge/models/assessments/AssessmentTest.java

```diff
@@ -133,6 +133,7 @@ public void serializeRountrip() throws Exception {
         assertEquals(node.get("additionalMetadata").get("key1").textValue(), "value1");
         assertEquals(node.get("additionalMetadata").get("key2").textValue(), "value2");
         assertEquals(node.get("type").textValue(), "Assessment");
+        assertEquals(node.get("phase").textValue(), "draft");
```

Comment:
I'm a bit confused by how this works. Isn't the enum defined uppercase?

Reply:
Yes, but the json serializer is set to do enums as lowercase: https://github.com/Sage-Bionetworks/BridgeServer2/blob/4c7433da919f08c6be05b2e41f4141ebe9850a9d/src/main/java/org/sagebionetworks/bridge/json/BridgeObjectMapper.java#L77

Reply:
That's handy